Runtime/Core/SelectionPickerRendererStandard.cs

```diff
@@ -91,7 +80,16 @@ internal class SelectionPickerRendererStandard: ISelectionPickerRenderer
                 RenderTexture.active.width));
                 */
 #endif
+
+                // URP does not support replacement shaders or custom passes from code, so for now it is necessary to
+                // force the pipeline to built-in when rendering. In editor it may be possible to use Handles.DrawCamera
+                // to avoid disposing and re-assigning the pipeline, as the RenderEditorCamera function has some logic
+                // that switches rendering path if replacement shaders are in use, but I wasn't able to get that
+                // approach to work without also requiring that the drawing happen during a repaint event.
+                var currentRenderPipeline = GraphicsSettings.renderPipelineAsset;
+                GraphicsSettings.renderPipelineAsset = null;
                 renderCam.RenderWithShader(shader, tag);
+                GraphicsSettings.renderPipelineAsset = currentRenderPipeline;
```

Comment:
Would that also fix the problem we have in probuilder with vertex selection in URP ?

Reply:
Yes, it's a workaround until we can implement it properly

Reply:
This workaround does not appear to work around in 2020.1.2f1.